### ISSUE TITLE
Examples of scrollableAncestor and threshold

### ISSUE THREAD

**Issue by garrettmaring** [brigade/react-waypoint#88]
It would be great to have some demos of how to use those props effectively!

**Comment by trotzig** [comment]
Agreed! Though I think adding a little bit more to the README would also be enough.

**Comment by jmeas** [comment]
@trotzig if you need any help updating the README, just lmk and I'd be happy to help.

**Comment by trotzig** [comment]
That would be great @jmeas! If you end up changing/enhancing `threshold`, maybe you could do it as part of that PR?

**Comment by jmeas** [comment]
I didn't end up adding these to the `threshold` PR, but I've added it to my todo list to do this as a separate PR.

**Comment by jmeas** [comment]
@garrettmaring I've added some short guides for these props in #111, which might be of use to you (note that `threshold` has been split out into `topOffset` and `bottomOffset`). The guides describe a few situations where a developer might choose to use these props.

I think demos are a great idea, too. I've added it on my todo list to put together more demos for them. In the meantime, if you have any specific questions about proper usage, ask 'em here and I can try to help out.
